test/blackbox/Dummy.spec.ts

```diff
@@ -0,0 +1,57 @@
+import {promises as fs} from 'fs';
+
+import * as path from 'path';
+import { TypeScriptGenerator, JavaGenerator, JavaScriptGenerator } from '../../src';
+import { execCommand, generateModels, renderModels } from './utils/Utils';
+import { renderModelsToSeparateFiles } from './utils/Utils';
+const fileToGenerate = path.resolve(__dirname, './docs/dummy.json');
+describe('Dummy JSON Schema file', function() {
+  jest.setTimeout(100000);
+  describe('should be able to generate and compile Java', () => {
+    test('class', async function() {
+      const generator = new JavaGenerator();
+      const generatedModels = await generateModels(fileToGenerate, generator);
+      expect(generatedModels).not.toHaveLength(0);
+      const renderOutputPath = path.resolve(__dirname, './output/java');
+      await renderModelsToSeparateFiles(generatedModels, renderOutputPath);
+      const compileCommand = `javac ${path.resolve(renderOutputPath, '*.java')}`
+      await execCommand(compileCommand);
+    });
+  });
+
+  describe('should be able to generate and transpile TS', () => {
+    test('class', async function() {
+      const generator = new TypeScriptGenerator();
+      const generatedModels = await generateModels(fileToGenerate, generator);
+      expect(generatedModels).not.toHaveLength(0);
+      const renderOutputPath = path.resolve(__dirname, './output/ts/output.ts');
+      await renderModels(generatedModels, renderOutputPath);
+      const transpiledOutputPath = path.resolve(__dirname, './output/ts/output.js');
+      const transpileAndRunCommand = `tsc -t es5 ${renderOutputPath} && node ${transpiledOutputPath}`;
```

Comment:
We can use es6. Who currently use the es5 in node project?! 😄

Reply:
True, let's expand the tests later 😄